util/projectRequest.ts

```diff
@@ -3,6 +3,43 @@ import { paginateRest } from '@octokit/plugin-paginate-rest';
 import githubColorsFixture from '../data/githubColors.json';
 import { Project, ACMCommitteeTopics, GitHubColors } from './types';
 
+export async function getUclaOpenSource(): Promise<Project[]> {
+  const PaginatedOctokit = Octokit.plugin(paginateRest);
+  const octokit = new PaginatedOctokit();
+  const response = await octokit.paginate('GET /search/repositories', {
+    q: 'ucla-opensource+in:topics',
+    per_page: 100,
+  });
+
+  const filteredData = response.filter((repo) => !repo.archived);
+  const sortedData = filteredData.sort(
+    (a, b) =>
+      new Date(b.updated_at as string).getTime() - new Date(a.updated_at as string).getTime(),
+  );
+  return sortedData.map((repo) =>
```

Comment:
Similar issue to [this pr!](https://github.com/uclaacm/opensource/pull/97/files#r917319276)

This entire function can be replaced by 
```suggestion
  return sortedData.map((repo) => ({
      name: repo.name,
      description: repo.description ?? '',
      link: repo.homepage ?? '',
      repo: repo.html_url,
      lang: repo.language ?? '',
      topics: repo.topics ?? [],
      image: getImageFromTopics(repo.topics).image,
      alt: getImageFromTopics(repo.topics).alt,
    })
```